apps/esmtool/record.cpp

```diff
@@ -15,8 +15,8 @@ void printAIPackage(ESM::AIPackage p)
         std::cout << "    Distance: " << p.mWander.mDistance << std::endl;
         std::cout << "    Duration: " << p.mWander.mDuration << std::endl;
         std::cout << "    Time of Day: " << (int)p.mWander.mTimeOfDay << std::endl;
-        if (p.mWander.mShouldRepeat != 1)
-            std::cout <<  "    Should repeat: " << (bool)p.mWander.mShouldRepeat << std::endl;
+        if (!p.mWander.mShouldRepeat)
+            std::cout <<  "    Should repeat: " << p.mWander.mShouldRepeat << std::endl;
```

Comment:
Oh, I see you changed the declaration too. But I suggest to change it back. None of the other esm structs are using bools. We need to be explicit about the size of the struct, so unsigned char is better style, I think.